src/transactions/feed/detailContent/NetworkFeeRowItem.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import TokenDisplay from 'src/components/TokenDisplay'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Fee, TransactionStatus } from 'src/transactions/types'
+
+export interface Props {
+  fees: Fee[]
+  transactionStatus: TransactionStatus
+}
+
+export default function NetworkFeeRowItem({ fees, transactionStatus }: Props) {
```

Comment:
Is that correct that we call this `Item` and are rendering only a single row, while the props are accepting the whole list (`Fee[]`)? It appears not particularly consistent, but maybe there is a smart plan behind the scenes?

Reply:
i don't really have a smart plan but my idea is to make this row reusable at least for all the transaction details. maybe then we can move it back into the main screen component. for now, this is only shared between approvals and swaps - i'll work with Nitya to make the various transaction details more consistent

Reply:
I was confused that it renders only a first item of the array. What if instead of passing the whole fees array in the props, we pass just a single fee item (the very same it will render)?

Reply:
i passed the whole fees array here because then consumers don't need to know that the network fee is the first element of the fees array. otherwise every transaction feed item needs to have the logic `fees[0] ? <NetworkFee /> : null`. it also didn't seem unreasonable to assume that we might one day change the network fees to be a different element index or a combination of the elements in the fees array...